sdk/modelsrepository/Azure.Iot.ModelsRepository/README.md

```diff
@@ -0,0 +1,89 @@
+# Azure IoT Models Repository client library for .NET
+
+This library provides functionality for interacting with the Azure IoT Models Repository. It also aims to provide a consistent experience working with digital twin model repositories following Azure IoT conventions.
+
+[Source code][source] | [Package (nuget)][package]
+
+## Getting started
+
+The complete Microsoft Azure SDK can be downloaded from the [Microsoft Azure downloads][microsoft_sdk_download] page, and it ships with support for building deployment packages, integrating with tooling, rich command line tooling, and more.
+
+For the best development experience, developers should use the official Microsoft NuGet packages for libraries. NuGet packages are regularly updated with new functionality and hotfixes.
+
+### Install the package
+
+Install the Azure IoT Models Repository client library for .NET with [NuGet][nuget]:
+
+```PowerShell
+Install-Package Azure.Iot.ModelsRepository
```

Comment:
Did the package name change with Azad's recent rename?

Reply:
only the client name changed, the namespace and project name remained the same.

Reply:
Why not also update the package name?

Reply:
If I download a package, I generally expect to add a using statement that matches it.

Reply:
We don't want `client` to be part of the package name do we?

Reply:
Namespace should equal package name. The client name itself wouldn't be part of the package name.

Reply:
My misread from earlier. It looks like we're good.